gematria/datasets/convert_bhive_to_llvm_exegesis_input.cc

```diff
@@ -60,43 +60,50 @@ int main(int argc, char* argv[]) {
     return 1;
   }
 
-  std::string initialRegValStr;
+  std::string initial_reg_val_str =
+      gematria::ConvertHexToString(kInitialRegVal);
+  std::string initial_mem_val_str =
+      gematria::ConvertHexToString(kInitialMemVal);
   {
-    // initialize initialRegValStr with kInitialRegValue in hex
-    std::stringstream ss;
-    ss << std::hex << kInitialRegVal << std::dec;
-    ss >> initialRegValStr;
-  }
-  std::string initialMemValStr;
-  {
-    // initialize initialMemValStr with kInitialMemValue in hex
+    // initialize initial_mem_val_str with kInitialMemValue in hex
     std::stringstream ss;
     ss << std::hex << kInitialMemVal << std::dec;
-    ss >> initialMemValStr;
+    ss >> initial_mem_val_str;
   }
```

Comment:
Is this part still needed? This should be handled by the `ConvertHexTostring()` on the line above, no?